docs/management/admin/host-isolation-ov.asciidoc

```diff
@@ -59,15 +61,21 @@ To release a host from a case alert:
 . Go to *Investigate -> Cases*, then click on the appropriate case to view the case activity its details.
 . Find the appropriate alert, then click the *Show alert details* button (*>*). The alert details flyout opens.
 . From the alert details flyout, click *Take action -> Release host*.
+. Enter a comment describing why you're releasing the host (optional).
+. Click *Confirm*.
 
 To release a host from an endpoint:
 
-. Go to *Manage -> Endpoints*, then select the appropriate host name in the *Endpoint* column. The endpoint details flyout opens.
-. Click *Take action -> Release host*.
+. Go to *Manage -> Endpoints*, then either:
+    * Select the appropriate endpoint in the *Endpoint* column, then click *Take action -> Release host* in the endpoint details flyout.
+    * Click the *Actions* menu (*...*) on the appropriate endpoint, then select *Release host*.
+. Enter a comment describing why you're releasing the host (optional).
+. Click *Confirm*.
 
-After the host is successfully released, the *Isolated* status is removed from the *Agent Status*.
+After the host is successfully released, the *Isolated* status is removed from the *Agent Status*. Any user currently on the host receives a notification that the computer has been reconnected to the network:
```

Comment:
```suggestion
After the host is successfully released, the *Agent Status* changes back to *Healthy*. Any user currently on the host receives a notification that the computer has been reconnected to the network:
```
I guess it could change back to something else but I don't think that would confuse anyone.

Reply:
@benironside This isn't quite correct; the "Isolated" label doesn't replace anything or vice versa; it's added or removed. So an endpoint could be both "Healthy" and "Isolated" at the same time (per the screenshot at the top of the page). But I agree that the phrasing is still inexact, so:
```suggestion
After the host is successfully released, the *Isolated* status is removed from the endpoint. Any user currently on the host receives a notification that the computer has been reconnected to the network:
```